packages/esm-patient-banner-app/src/contact-details/contact-details.scss

```diff
@@ -33,20 +33,23 @@
 .relationship div {
   flex: 1 1;
 }
+
 .verticalLine {
   position: relative;
 }
+
 .verticalLine:after {
   content: '';
   position: absolute;
   border-left: $spacing-01 solid $ui-03;
-  left:50%;
+  left: 50%;
   top: 0;
   bottom: 0;
 }
+
 .hr{
   height: $spacing-01;
-  background-color:$ui-03;
+  background-color: $ui-03;
```

Comment:
thanks @KorirC   just curious to see whether the two statements have a difference, could you be having any quick snippet  before the changes